js/aax.js

```diff
@@ -3238,6 +3238,46 @@ module.exports = class aax extends Exchange {
         };
     }
 
+    async fetchTransactionFees (codes = undefined, params = {}) {
+        /**
+         * @method
+         * @name aax#fetchTransactionFees
+         * @description fetch transaction fees
+         * @see https://www.aax.com/apidoc/index.html#get-currencies
+         * @param {[string]|undefined} codes list of unified currency codes
+         * @param {object} params extra parameters specific to the aax api endpoint
+         * @returns {object} a list of [fee structures]{@link https://docs.ccxt.com/en/latest/manual.html#fee-structure}
+         */
+        await this.loadMarkets ();
+        const currencyKeys = Object.keys (this.currencies);
+        const result = {};
+        for (let i = 0; i < currencyKeys.length; i++) {
+            const code = currencyKeys[i];
```

Comment:
I think that we should start using a [`parseTransactionFees`](https://github.com/ccxt/ccxt/pull/15574/files#r1014869320) and [`parseTransactionFee`](https://github.com/ccxt/ccxt/pull/15574/files#r1014869330) method, like how the mexc PR does it